packages/frontend/src/components/common/modal/DashboardCreateModal.tsx

```diff
@@ -118,111 +109,81 @@ const DashboardCreateModal: FC<DashboardCreateModalProps> = ({
             {...modalProps}
             onClose={handleClose}
         >
-            <DialogBody>
-                <FormGroup label="Name your dashboard" labelFor="chart-name">
-                    <InputGroup
-                        id="chart-name"
-                        type="text"
-                        value={dashboardName}
-                        onChange={(e) => setDashboardName(e.target.value)}
-                        placeholder="eg. KPI dashboard"
+            <Form
+                title="Create Dashboard"
+                methods={form}
+                onSubmit={handleConfirm}
+            >
+                <DialogBody>
+                    <Input
+                        label="Enter a memorable name for your dashboard"
+                        name="dashboardName"
+                        placeholder="eg. KPI Dashboards"
+                        disabled={isCreatingDashboard}
```

Comment:
here we should also take whatever `useForm` gives us and

```
 {...form.register('dashboardName')}
```
And remove the `name` prop
What do you think?